### ISSUE TITLE
search results video preview autoplays/displays media icon 

### ISSUE THREAD

**Issue by LaurenWags** [brave/browser-android-tabs#479]
**Did you search for similar issues before submitting this one?**
Yes

**Description:**
Reported via community:

When google searching for something that returns video results, a preview autoplays (even with autoplay off). This triggers brave to give the notification that media is playing. The notifications can be paused and swiped away, but also persist after switching tabs or apps.

**Device Details:**
Reproduced using the following
   - Install Type(ARM, x86): ARM
   - Device(Phone, Tablet, Phablet): Galaxy S8
   - Android Version: 7.0
    
**Brave Version:**
1.0.37

**Steps to reproduce:**
1. Open Brave, go to Settings > Site Settings > Media. Turn off Autoplay.
2. Open a new tab (Google is the default search engine).
3. Search for 'brave walkthrough’
4. Tab displays search results.
5. Wait a couple of seconds, the first video in the list will do an auto preview.
6. Up in the notifications bar, see the media icon (speaker).
7. Search again for 'brave videos’
8. Tab displays search results.
9. No videos on page show an auto preview.
10. No media icon is displayed in the notifications bar.

**Actual Behavior**
preview autoplays, media icon is displayed

**Expected Behavior**
preview should not autoplay since it is off and no media icon (speaker) should be displayed

**Is this an issue with Beta build?**
No

**Is this an issue in the currently released version?**
Yes

**Can this issue be consistently reproduced?**
Fairly easily

**Extra QA steps:**
    1. 
    2. 
    3.

**Website problems only:**
  - did you check with Brave Shields down?
  - did you check in Chrome for same behavior? yes, here's what happens when using Chrome:
1. Open Chrome, go to Settings > Site Settings > Media. Turn off Autoplay.
2. Open a new tab (Google is the search engine).
3. Search for 'brave walkthrough’
4. Tab displays search results.
5. Wait a couple of seconds, the first video in the list will do an auto preview.
6. However, up in the notifications bar, no media icon is displayed.

**Screenshot if needed:**
![previewautoplay](https://user-images.githubusercontent.com/28145373/33728814-b1ad5c4a-db49-11e7-90a6-9714bf80b2fd.jpg)

**Any related issues:**

**Comment by LaurenWags** [comment]
Multiple +1s on reported community thread.

**Comment by LaurenWags** [comment]
@SergeyZhukovsky @samartnik this issue is coming up more, could you guys take a look and see if it could be prioritized?

**Comment by samartnik** [comment]
Can't reproduce this on my devices. @LaurenWags are you able to reproduce this?

**Comment by LaurenWags** [comment]
@samartnik yes but I had to tweak my steps from when this was first reported. Steps I used are:
(you can start playing music at this point to see why this is so annoying to users, I used Spotify app to play music on my phone)
1. Using 1.0.48 Brave, go to Settings > Site Settings > Media. Turn off Autoplay if enabled.
2. Open a new tab and navigate to google.com (must do this, searching thru search bar does not produce this issue any more)
3. Search for 'brave’ in the search box on page.
4. page displays search results.
5. Scroll down the page and stop when you see clips from the movie Brave. Wait a couple of seconds, the first video in the list will do an auto preview.
6. Up in the notifications bar, see the media icon (speaker) and your music will stop playing.
7. Search again for 'brave videos’
8. Page displays search results.
9. No videos on page show an auto preview.
10. No media icon is displayed in the notifications bar, your music is not interrupted.

**Comment by msx-tr** [comment]
OP here, issue is still ongoing and when it occurs, it also pauses any active media playback

ie: have Spotify playing, peruse a query that invokes video results, autoplay video tile begins which gives playback icon in notification area and pauses Spotify playback

Currently using brave external version 1.0.52, Note 8 VZW, Android: 8.0

**Comment by Brave-Matt** [comment]
+1 from support@brave.com email
User's message: 

> Hi,
> I have a question regarding Brave, the Android app. Despite having disabled the video autostart they just keep starting by themselves. Have a set the settings wrong?
> Best regards
